notebooks/spark-intro.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "95ca3df2-cad2-4e16-8768-bfca026bb576",
+   "metadata": {},
+   "source": [
+    "# Spark RDDs\n",
+    "This notebook is intended to teach you the basic structure of Apache Spark, from the basic structures to some of the more advanced techniques.  We'll do our best to give you practical advice, and not get bogged down in little minutia."
```

Comment:
Edit: "This notebook is intended to teach you Apache Spark; from the basic structures to a selection of the more advanced techniques. We'll do our best to give you practical advice, and not get bogged down in minutia."